packages/ui/src/components/EventsCard/EventsCard.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { EventCardProps, EventsCard } from './EventsCard';
+import { EVENTS as events } from './EventsCard.mocks';
+
+const meta: Meta = {
+  title: 'Components/EventsCard',
+  component: EventsCard,
```

Comment:
💭 You can add `width` to the controls list and change its type to a number so it would be easier to customize it

Reply:
it's done